src/WelcomeView.ts

```diff
@@ -0,0 +1,16 @@
+import WelcomeViewTemplate from 'html-loader!./welcome_view.html';
+
+export interface IWelcomeView {
+  build();
+}
+
+export default class WelcomeView implements IWelcomeView {
+
+  constructor(private parent: HTMLElement) {
+    //
+  }
+
+  build() {
+    this.parent.innerHTML = WelcomeViewTemplate;
```

Comment:
is the `build` method on the WelcomeView needed? wouldn't it be sufficient to set `this.parent.innerHTML` in the constructor?

Reply:
Yes, you are right. My idea was to provide an interface for the welcome views since they are distributed across multiple repositories. And Typescript does not allow empty interfaces. That is the reason for the `build()` method.